### ISSUE TITLE
kedro info | install | build-reqs returns TypeError

### ISSUE THREAD

**Issue by niquet** [quantumblacklabs/kedro#890]
## Description
After running kedro new and switching to the project directory, all kedro commands return the following error: 
TypeError: _validate_items() got an unexpected keyword argument 'only'.

## Context
As of right now I cannot continue with the exciting journey of getting to know through the tutorials.

## Steps to Reproduce
1. Create a new project with ```kedro new```
2. Switch into the project directory
3. Install project dependencies with ```kedro install```

## Expected Result
Dependencies should be installed.

## Actual Result
Receiving TypeError. Actual file path has been masked by me.

```
File ".../spaceflights/venv/bin/kedro", line 10, in <module>
    sys.exit(main())
  File ".../spaceflights/venv/lib/python3.7/site-packages/kedro/framework/cli/cli.py", line 264, in main
    cli_collection = KedroCLI(project_path=Path.cwd())
  File ".../spaceflights/venv/lib/python3.7/site-packages/kedro/framework/cli/cli.py", line 182, in __init__
    self._metadata = bootstrap_project(project_path)
  File ".../spaceflights/venv/lib/python3.7/site-packages/kedro/framework/startup.py", line 184, in bootstrap_project
    configure_project(metadata.package_name)
  File ".../spaceflights/venv/lib/python3.7/site-packages/kedro/framework/project/__init__.py", line 219, in configure_project
    settings.configure(settings_module)
  File ".../spaceflights/venv/lib/python3.7/site-packages/dynaconf/base.py", line 182, in configure
    self._wrapped = Settings(settings_module=settings_module, **kwargs)
  File ".../spaceflights/venv/lib/python3.7/site-packages/dynaconf/base.py", line 236, in __init__
    only=self._validate_only, exclude=self._validate_exclude
  File ".../spaceflights/venv/lib/python3.7/site-packages/dynaconf/validator.py", line 417, in validate
    validator.validate(self.settings, only=only, exclude=exclude)
  File ".../spaceflights/venv/lib/python3.7/site-packages/dynaconf/validator.py", line 198, in validate
    settings, settings.current_env, only=only, exclude=exclude
TypeError: _validate_items() got an unexpected keyword argument 'only'
```

## Your Environment
Include as many relevant details about the environment in which you experienced the bug:

* Kedro version used (`pip show kedro` or `kedro -V`): 0.17.4
* Python version used (`python -V`): Python 3.7.4
* Operating system and version: macOS Catalina Version 10.15.7

**Comment by datajoely** [comment]
Hi this is a duplicate of this issue https://github.com/quantumblacklabs/kedro/issues/888 - we have an interim fix and a proper fix will be released shortly in 0.17.5

**Comment by niquet** [comment]
@datajoely Thank you so much for the quick response!
